Explorer++/Helper/FileOperations.cpp

```diff
@@ -649,3 +671,77 @@ void FileOperations::DeleteFileSecurely(const std::wstring &strFilename,
 
 	DeleteFile(strFilename.c_str());
 }
+
+HRESULT FileOperations::Undelete(const PCIDLIST_ABSOLUTE &pidl)
+{
+	wil::com_ptr_nothrow<IShellFolder> pDesktop{};

```

Comment:
There's not really any need for the braces at the end of this line and for consistency with the other code, it would be better to omit them.